dslink-scala-nest/src/main/scala/com/uralian/nest/model/HvacState.scala

```diff
@@ -23,7 +23,9 @@ object HvacState {
     * @return
     */
   def forName(name: String): HvacState = name.toLowerCase match {
+    case "heating" => Heat
```

Comment:
should those be instead of "heat" and "cool", or they are additional states? not sure if i missed them in the docs... there is "HvacState" and "HvacMode", want to make sure we set them right.

Reply:
Json parsing was breaking because of this, I am not sure if those come with different names from stream than from pulling API. Let's check